backend/app/utils/ai_processors.py

```
@@ -331,3 +269,118 @@ def __init__(self):
         super().__init__(
             template_name="key_points.j2", processor_name="key_points_extractor"
         )
+
+async def analyze_content_with_ai(
+    content_item: ContentItem,
+    session: Session,
```

Ruff (0.11.9)

Undefined name `Session`

(F821)